OCP pre-release nerfs

astralAU opened the pull request:
Changes to:
28565	Perverse Demon Mask [1]
28531	Blacksmith's Gloves [1]
19139	Survivor's Orb
20931	Prism Ranger Scarf
28502	Mob Scarf
28594	Temporal Ring [1]

<!-- NOTE: Anything within these brackets will be hidden on the preview of the Pull Request. -->

* **Addressed Issue(s)**: 

<!--
Please specify the ChocobotRO [GitHub issue(s)](https://help.github.com/articles/autolinked-references-and-urls/#issues-and-pull-requests) this pull request amends.
If no issue exists yet, please [create one](https://github.com/Latiosu/chocobotro/issues/new) first and then link your pull request to the amendment!
-->

* **Description of Pull Request**: 

<!-- Describe how this pull request will resolve the issue(s) listed above. -->

astralAU committed:
OCP pre-release nerfs

28565	Perverse Demon Mask [1]
28531	Blacksmith's Gloves [1]
19139	Survivor's Orb
20931	Prism Ranger Scarf
28502	Mob Scarf
28594	Temporal Ring [1]

astralAU committed:
Fixed syntax error on Blacksmith's Glove

Latiosu commented:
Thanks for creating this @gruberaj.
I'll be merging it into a separate branch before bringing it into the default branch.

gruberaj commented:
> 
> 
> Thanks for creating this @gruberaj.
> I'll be merging it into a separate branch before bringing it into the default branch.

Agreed, this way should you need to revert the changes and remain, revert to default, change the nerfs etc we can do so separately.

astralAU committed:
Nerfed Magician's Glove (equivalent to Blacksmith's Glove)

This was missed during our 2nd review. 50% MDEF ignore is far too strong for 1 item, 2 of these equates to 100%. Nerfed to 25% so 50% max between two. As mentioned in title, same treatment as Blacksmith's Glove which has been heavily reduced as well (50% to 20%).

latiosu commented on db/re/item_db_equip.yml:
```diff
@@ -99681,7 +99681,7 @@ Body:
     EquipLevelMin: 80
     Script: |
-      bonus bDelayrate,-10; bonus2 bAddClass,Class_Boss,5; bonus2 bIgnoreDefRaceRate,RC_All,50;
+      bonus bDelayrate,-10; bonus2 bAddClass,Class_Boss,5; bonus2 bIgnoreDefRaceRate,RC_All,20; bonus2 bIgnoreDefRaceRate,RC_Player_human,-20;
```
```suggestion
      bonus bDelayrate,-10;
      bonus2 bAddClass,Class_Boss,5;
      bonus2 bIgnoreDefRaceRate,RC_All,20;
      bonus2 bIgnoreDefRaceRate,RC_Player_Human,-20;
```

latiosu commented on db/re/item_db_equip.yml:
```diff
@@ -99804,7 +99804,7 @@ Body:
     EquipLevelMin: 80
     Script: |
-      .@s = readparam(bStr); .@a = readparam(bAgi); .@d = readparam(bDex); .@v = readparam(bvit); .@l = readparam(bLuk); .@i = readparam(bInt); bonus bStr,3*(.@i/18); bonus bAgi,3*(.@l/18); bonus bVit,3*(.@d/18); bonus bInt,3*(.@s/18); bonus bDex,3*(.@v/18); bonus bLuk,3*(.@a/18); bonus bMaxHPrate,(.@d/18); bonus bFlee2,(.@a/18); bonus bMaxHPrate,(.@d/18); bonus bVariableCastrate,-(.@v/18); bonus ...
+      .@s = readparam(bStr); .@a = readparam(bAgi); .@d = readparam(bDex); .@v = readparam(bvit); .@l = readparam(bLuk); .@i = readparam(bInt); bonus bStr,3*(.@i/18); bonus bAgi,3*(.@l/18); bonus bVit,3*(.@d/18); bonus bInt,3*(.@s/18); bonus bDex,3*(.@v/18); bonus bLuk,3*(.@a/18); bonus bMaxHPrate,(.@d/18); bonus bFlee2,(.@a/18); bonus bMaxHPrate,(.@d/18); bonus bVariableCastrate,-(.@v/18); bonus ...
```
```suggestion
      .@s = readparam(bStr);
      .@a = readparam(bAgi);
      .@d = readparam(bDex);
      .@v = readparam(bvit);
      .@l = readparam(bLuk);
      .@i = readparam(bInt);
      bonus bStr,3*(.@i/18);
      bonus bAgi,3*(.@l/18);
      bonus bVit,3*(.@d/18);
      bonus bInt,3*(.@s/18);
      bonus bDex,3*(.@v/18);
      bonus bLuk,3*(.@a/18);
      bonus bMaxHPrate,(.@d/18);
      bonus bFlee2,(.@a/18);
      bonus bMaxHPrate,(.@d/18);
      bonus bVariableCastrate,-(.@v/18);
      bonus bDelayrate,-(.@i/18);
      if (BaseLevel>=100) {
          bonus2 bIgnoreMdefRaceRate,RC_All,8*(.@s/18);
          bonus2 bIgnoreMdefRaceRate,RC_Player_Human,-8*(.@s/18);
      }
      if (BaseLevel>=100) {
          bonus2 bIgnoreDefRaceRate,RC_All,5*(.@i/18);
          bonus2 bIgnoreDefRaceRate,RC_Player_Human,-5*(.@i/18);
      };
```

latiosu reviewed (changes requested):
Looks great just a couple script changes necessary
